usim.py

```diff
@@ -384,7 +384,8 @@ def monitor_sub(q, index):
 	t0 = time.time()
 	log('Started main thread %d' % (index+1))
 	while True:
-		log("Restarting loop", console_only=True)
+		#log("Restarting loop", console_only=True)
```

Comment:
Yeah, I try to avoid leaving commented code in there, but I make an exception for debug code like `log` calls.

As for commenting out the "Restarting loop..." line, I decided against that. I use `screen` to run it on the backend, so the extra print calls mean nothing unless I've attached to that screen. And since the prints are timestamped, it's easy to tell if the script is still running, and if not, exactly when it failed. A heartbeat, basically.